src/frontend/screens/WineManager/index.tsx

```diff
@@ -113,10 +127,7 @@ export default React.memo(function WineManager(): JSX.Element | null {
             {!refreshing &&
               !!wineVersions.length &&
               wineVersions.map((release, key) => {
-                if (release.type === repository) {
```

Comment:
I'm moving the filter when setting the array so we don't have to loop through all the elements in every render